ibm-colors.json

```diff
@@ -1,172 +0,0 @@
-{
```

Comment:
I'm pretty sure people are using this file by creating scss maps with imports. I would like to leave the assets (including the old _ibm-colors.scss) in place and we'll remove them in a 2.0.0 release once we generate the dist at release time.